src/com/boxboat/jenkins/library/buildVersions/GitBuildVersions.groovy

```diff
@@ -54,12 +54,14 @@ class GitBuildVersions implements Serializable {
 
         def dir = "${gitRepo.dir}/image-versions/${event}"
         def path = "${dir}/${Utils.alphaNumericDashLower(image.path)}.txt"
-        def key = "image_tag_${Utils.alphaNumericUnderscoreLower(image.path)}"
+        def tag_key = "image_tag_${Utils.alphaNumericUnderscoreLower(image.path)}"
+        def info_key = "image_info_${Utils.alphaNumericUnderscoreLower(image.path)}"
```

Comment:
Maybe I should change this to `image_event_tag_*`

Reply:
I think that `event_key` is a better variable name and `image_event_${...}` is a better yaml/env var name

Reply:
👍